public/assets/images/ComponentDefaultImage/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ComponentDefaultImage(props) {
```

Comment:
This is not an asset this is a react component (js code), here goes the assets (the svg directly like the other images here).
Also the file name does not need to begin with capital letter.